_health-care/_js/components/financial-assessment/SpouseInformationSection.jsx

```diff
@@ -37,6 +39,57 @@ class SpouseInformationSection extends React.Component {
       );
     }
 
+    if (!this.props.data.sameAddress) {
+      spouseAddressSummary = (
```

Comment:
If the spouse and the veteran have the same address is it fine that we just don't show the address or do we need to update the `spouseAddress` fields with the veteran `address`?